src/templates/homepage.html

```diff
@@ -0,0 +1,25 @@
+<div class="app-preview"></div>
+
+<div class="app-list">
+  {% defer (url=api('apps'), pluck='objects', as='apps', key='slug') %}
+    <ul class="clearfix">
+      {% for app in this %}
+        <li class="app-list-app focusable" data-slug="{{ app.slug }}"
+            contextmenu="contextmenu-{{ loop.index }}">
+          <img src="{{ app.icons['128'] }}"
+               alt="{{ app.name }}" title="{{ app.name }}">
+          <span class="name">{{ app.name }}</span>
+
+          <menu type="context" id="contextmenu-{{ loop.index }}">
```

Comment:
Is this enough to add context? I don't see the associated JS or is the OS handling that?